.github/workflows/test-nx-project-e2e.yaml

```diff
@@ -0,0 +1,123 @@
+# This file is used for tests that use Nx app containers.
+# For tests that don't require containers to be built first, see test-code.yaml
+
+name: Test BCIERS Nx App containers
+
+on:
+  workflow_call:
+    inputs:
+      nx_app_port:
+        description: "Port to run the Nx app on"
+        required: true
+        type: number
+      nx_project:
+        description: "Nx project to test"
+        required: true
+        type: string
+      image_url:
+        description: "Docker image URL minus the tag which will be the pull request sha"
+        required: true
+        type: string
+
+env:
+  PGUSER: postgres
+  DJANGO_SECRET_KEY: ${{ secrets.DJANGO_SECRET_KEY }}
+  KEYCLOAK_CLIENT_ID: ${{ secrets.KEYCLOAK_CLIENT_ID }}
+  KEYCLOAK_CLIENT_SECRET: ${{ secrets.KEYCLOAK_CLIENT_SECRET }}
+  NEXTAUTH_SECRET: ${{ secrets.NEXTAUTH_SECRET }}
+
+jobs:
+  e2e-tests:
+    name: 🧪 e2e tests ${{ matrix.project }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - project: chromium
+            os: ubuntu-24.04
+            cache_dir: ~/.cache/ms-playwright
+          - project: firefox
+            os: ubuntu-latest
+            cache_dir: ~/.cache/ms-playwright
+          - project: webkit
+            os: macos-latest
+            cache_dir: ~/Library/Caches/ms-playwright
```

Comment:
Not sure if we want to test both these browsers unlike reg1 which has these commented out due to flakey issues.